src/test/java/BaseTest.java

```diff
@@ -1,11 +1,68 @@
 import io.github.bonigarcia.wdm.WebDriverManager;
+import io.netty.util.Timeout;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
 import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Parameters;
+
+import java.time.Duration;
 
 public class BaseTest {
+    @BeforeMethod
```

Comment:
Define this static variable inside BaseTest class:
    `public static WebDriver driver = null;`
    `public static String url = null;`
   `public static final ThreadLocal<WebDriver> threadDriver = new ThreadLocal<>();`